pages/api/recipes/index.js

```diff
@@ -0,0 +1,26 @@
+import Recipe from "../../../components/db/models/Recipe";
+import dbConnect from "../../../components/db/connect";
+
+export default async function handler(request, response) {
+  await dbConnect();
+  if (request.method === "GET") {
+    const recipes = await Recipe.find();
+    response.status(200).json(recipes);
+    return;
+  }
+  if (request.method === "POST") {
+    try {
+      const recipe = request.body;
+      const newRecipe = new Recipe(recipe);
+      await newRecipe.save();
+      response.status(200).json({ success: true, data: newRecipe });
```

Comment:
Success is implicit from the 200 code ^^

It should be [201 Created](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201) actually!